these install /usr/lib/qt6/plugins/multimedia/libmockmultimediaplugin.a which
then has to exist for the -devel to work, but not splitting static keeps around
stuff that can be split, so just don't build this instead
--
--- a/tests/CMakeLists.txt
+++ b/tests/CMakeLists.txt
@@ -7,4 +7,3 @@
     # Add qt_find_package calls for extra dependencies that need to be found when building
     # the standalone tests here.
 endif()
-qt_build_tests()
